test/plugin/constants.rb

```diff
@@ -265,6 +269,8 @@ module Constants
       }
     )
   ).freeze
+  COMPUTE_PARAMS_WITH_MULTI_TAGS =
```

Comment:
That's doable, but it will require touching many tests. If we decide to do it, I'll make the change in a separate PR.

```
grep -r COMPUTE_PARAMS .
./test/plugin/base_test.rb:    verify_log_entries(1, COMPUTE_PARAMS)
./test/plugin/base_test.rb:    verify_log_entries(1, COMPUTE_PARAMS.merge(
./test/plugin/base_test.rb:    verify_log_entries(1, COMPUTE_PARAMS, 'jsonPayload') do |entry, i|
./test/plugin/base_test.rb:    verify_log_entries(1, COMPUTE_PARAMS, 'jsonPayload') do |entry|
./test/plugin/base_test.rb:    verify_log_entries(3, COMPUTE_PARAMS, 'textPayload') do
./test/plugin/base_test.rb:    verify_log_entries(6, COMPUTE_PARAMS, 'jsonPayload') do |entry|
./test/plugin/base_test.rb:    verify_log_entries(1, COMPUTE_PARAMS, 'textPayload') do
./test/plugin/base_test.rb:    verify_log_entries(2, COMPUTE_PARAMS, 'jsonPayload') do |entry|
./test/plugin/base_test.rb:    verify_log_entries(6, COMPUTE_PARAMS, 'jsonPayload') do |entry|
./test/plugin/base_test.rb:      verify_log_entries(0, COMPUTE_PARAMS, 'jsonPayload')
./test/plugin/base_test.rb:      verify_log_entries(1, COMPUTE_PARAMS.merge(log_name: encoded_tag),
./test/plugin/base_test.rb:      verify_log_entries(1, COMPUTE_PARAMS.merge(log_name: sanitized_tag),
./test/plugin/base_test.rb:      verify_log_entries(log_entries_count, COMPUTE_PARAMS_WITH_MULTI_TAGS,
./test/plugin/base_test.rb:        verify_log_entries(emit_index, COMPUTE_PARAMS) do |entry, i|
./test/plugin/base_test.rb:    verify_log_entries(1, COMPUTE_PARAMS, 'jsonPayload') do |entry|
./test/plugin/base_test.rb:    verify_log_entries(1, COMPUTE_PARAMS)
./test/plugin/base_test.rb:    # make a deep copy of COMPUTE_PARAMS and add the parsed label.
./test/plugin/base_test.rb:    params = Marshal.load(Marshal.dump(COMPUTE_PARAMS))
./test/plugin/base_test.rb:    # make a deep copy of COMPUTE_PARAMS and add the parsed label.
./test/plugin/base_test.rb:    params = Marshal.load(Marshal.dump(COMPUTE_PARAMS))
./test/plugin/base_test.rb:    # make a deep copy of COMPUTE_PARAMS and add the parsed label.
./test/plugin/base_test.rb:    params = Marshal.load(Marshal.dump(COMPUTE_PARAMS))
./test/plugin/base_test.rb:    # make a deep copy of COMPUTE_PARAMS and add the parsed labels.
./test/plugin/base_test.rb:    params = Marshal.load(Marshal.dump(COMPUTE_PARAMS))
./test/plugin/base_test.rb:      verify_log_entries(n, COMPUTE_PARAMS)
./test/plugin/base_test.rb:      verify_log_entries(1, COMPUTE_PARAMS, 'httpRequest') do |entry|
./test/plugin/base_test.rb:      verify_log_entries(1, COMPUTE_PARAMS, 'httpRequest') do |entry|
./test/plugin/base_test.rb:      verify_log_entries(1, COMPUTE_PARAMS, 'httpRequest') do |entry|
./test/plugin/base_test.rb:      verify_log_entries(1, COMPUTE_PARAMS, 'jsonPayload') do |entry|
./test/plugin/base_test.rb:    verify_log_entries(1, COMPUTE_PARAMS, destination_key) do |entry|
./test/plugin/base_test.rb:    verify_log_entries(1, COMPUTE_PARAMS, destination_key) do |entry|
./test/plugin/base_test.rb:    verify_log_entries(1, COMPUTE_PARAMS, 'jsonPayload') do |entry|
./test/plugin/constants.rb:  COMPUTE_PARAMS = {
./test/plugin/constants.rb:  COMPUTE_PARAMS_WITH_METADATA_VM_ID_AND_ZONE = COMPUTE_PARAMS.merge(
./test/plugin/constants.rb:    resource: COMPUTE_PARAMS[:resource].merge(
./test/plugin/constants.rb:  COMPUTE_PARAMS_WITH_MULTI_TAGS =
./test/plugin/constants.rb:    COMPUTE_PARAMS.reject { |k, _| k == :log_name }.freeze
./test/plugin/test_out_google_cloud.rb:    verify_log_entries(emit_index, COMPUTE_PARAMS) do |entry|
./test/plugin/test_out_google_cloud.rb:      verify_log_entries(1, COMPUTE_PARAMS) do |entry|
./test/plugin/test_out_google_cloud_grpc.rb:    verify_log_entries(emit_index, COMPUTE_PARAMS) do |entry|
./test/plugin/test_out_google_cloud_grpc.rb:    verify_log_entries(1, COMPUTE_PARAMS, 'jsonPayload') do |entry|
./test/plugin/test_out_google_cloud_grpc.rb:      verify_log_entries(1, COMPUTE_PARAMS) do |entry|
```

Reply:
I see. Yes, probably no point in switching names now, but you could at least redefine these in an additive way rather than use `reject`, i.e.:
```ruby
   COMPUTE_PARAMS_NO_LOG_NAME = {
     resource: {
       type: COMPUTE_CONSTANTS[:resource_type],
       labels: {
         'instance_id' => VM_ID,
         'zone' => ZONE
       }
     },
     project_id: PROJECT_ID,
     labels: {
       "#{COMPUTE_CONSTANTS[:service]}/resource_name" => HOSTNAME
     }
   }.freeze
   COMPUTE_PARAMS_WITH_METADATA_VM_ID_AND_ZONE = COMPUTE_PARAMS.merge(
     resource: COMPUTE_PARAMS[:resource].merge(
       labels: {
         'instance_id' => METADATA_VM_ID,
         'zone' => METADATA_ZONE
        }
      )
    ).freeze
    COMPUTE_PARAMS = COMPUTE_PARAMS_NO_LOG_NAME.merge(
      log_name: 'test'
    ).freeze
```

Reply:
Changed.